src/sources/db3/db3-schema.lisp

```diff
@@ -4,40 +4,24 @@
 
 (in-package :pgloader.source.db3)
 
-(defclass dbf-connection (fd-connection)
-  ((db3 :initarg db3 :accessor fd-db3))
-  (:documentation "pgloader connection parameters for DBF files."))
+(defclass copy-db3 (db-copy)
+  ((encoding    :accessor encoding	  ; file encoding
+	        :initarg :encoding))
```

Comment:
This line mixes tabs and spaces.

Reply:
It's my Emacs setup that does that, I think. And I think it's just standard SLIME setup here. In other words I don't mind too much about that.